docs/get-started/developers/index.md

```diff
@@ -47,9 +47,29 @@ Our design system libraries and the documentation website offer assets and guida
 
 ## About web components
 
-Web components are based on a set of web platform APIs that help to create reusable and encapsulated UI elements. Those standards consist of custom elements, shadow DOM, and HTML Templates.
+### What are Web Components?
 
-Because they're able to work with any framework that supports HTML, web components can be used without having to rework all of your code and are less likely to be affected by changes in preferred web stacks. Encapsulation within the shadow DOM prevents a page's code from breaking a component's style and allows for a scalable design system.
+Web components are based on a set of web platform APIs that help to create reusable and encapsulated UI elements. Those standards consist of [custom elements](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_custom_elements), [shadow DOM](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_shadow_DOM), and [HTML Templates](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_templates_and_slots).
```

Comment:
ce's don't neccessarily have to be reusable, or encapsulated

but they can be either or both

other standards: css custom properties, css shadow parts, ecmascript modules, element internals

Reply:
@bennypowers can you suggest a rephrasing of this?